ci/jenkins/jobs/projects.yaml

```diff
@@ -399,18 +406,5 @@
                 - text:
                     credential-id: GOVC_PASSWORD
                     variable: GOVC_PASSWORD
-      - '{test_name}-testbed-gc-on-all-nodes':
-          test_name: nephe
-          node: nephe-gc
-          triggers:
-            - timed: "H 7 * * *"
-          axes:
-            - axis:
-                type: slave
-                name: nodes
-                values:
-                  - nephe-test-1
-                  - nephe-test-2
-                  - nephe-test-3
           builders:
             - nephe-testbed-gc-on-all-nodes
```

Comment:
This builder should be nephe-gc-cronjob, and we can delete builder nephe-testbed-gc-on-all-nodes from macros.yaml.